app/views/users/new.html.erb

```diff
@@ -1,3 +1,2 @@
-<% provide(:title, 'Sign up') %>
-<h1>Sign up</h1>
-<p>This will be a signup page for new users.</p>
+<h1>Users#new</h1>
```

Comment:
From now please use I18n(internationalization) for you application. 
Reference: [http://guides.rubyonrails.org/i18n.html](http://guides.rubyonrails.org/i18n.html)